releng/releng-target/xtext-extras.target.target

```diff
@@ -3,15 +3,17 @@
 <target name="org.eclipse.xtext.extras.target" sequenceNumber="0">
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.26.0/"/>
+			<repository location="https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.28.0/"/>
 			<unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.xtend" version="0.0.0"/>
+			<unit id="org.eclipse.xpand" version="0.0.0"/>
```

Comment:
Typesystem emf ?

Reply:
https://github.com/eclipse/xtext-extras/blob/1c3f268d7f270a37850eeb0bdef39b47a33836b9/releng/releng-target/xtext-extras.target.target#L9

Reply:
Yeah. Missed that twice ...